faker/faker_test.go

```diff
@@ -46,3 +47,16 @@ func TestFakerWithCustomType(t *testing.T) {
 	assert.NotEmpty(t, a.C)
 	assert.NotEmpty(t, a.D)
 }
+
+type complexType struct {
+	IPAddress net.IP
+}
+
+func TestFakerWithComplexCustomType(t *testing.T) {
```

Comment:
```suggestion
func TestFakerCanFakeNet.IP(t *testing.T) {
```

[non blocking] Maybe a name that describes what we're testing in a more specific way